.github/workflows/docs-pr.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     permissions:
       contents: read
     name: Build Ansible Docs
-    uses: ansible-collections/community.hashi_vault/.github/workflows/_shared-docs-build-pr.yml@main
+    uses: ansible-community/github-docs-build/.github/workflows/_shared-docs-build-pr.yml@main
```

Comment:
I guess at some point it makes sense to pin the version to a tag, but right now while things are still in flux let's better keep `main` :)

Reply:
Looks like that's not really trivial: https://github.com/ansible-community/github-docs-build/issues/4

Reply:
yes, I think while we're getting things set up and ironing out early bugs and such, `main` is better. I consider this in "preview" basically, and anyone who'd like to use it can do so, and if it causes CI failures the entire workflow can be temporarily disabled/enabled in each repo that uses it (in the GitHub UI, commits are not needed). We'll come up with a release strategy at some point; https://github.com/ansible-community/github-docs-build/issues/4 does complicate the matter a bit, it's true.